onnxruntime/contrib_ops/cpu/bert/attention_base.h

```diff
@@ -37,6 +37,7 @@ class AttentionBase {
     num_heads_ = static_cast<int>(num_heads);
 
     is_unidirectional_ = info.GetAttrOrDefault<int64_t>("unidirectional", 0) == 1;
+    mask_filter_value_ = info.GetAttrOrDefault<float>("mask_filter_value", std::numeric_limits<float>::lowest());
```

Comment:
I think the default value can be -10000 for backward compatible. Assume that user have an old model, we shall make sure the result is same (old model does not have the attribute).